src/common/Web3Client.ts

```diff
@@ -1,15 +1,23 @@
 import Web3 from 'web3'
 import { SendOptions } from '../types/Common'
+import Biconomy from '@biconomy/mexa'
 
 export default class Web3Client {
   public parentWeb3: Web3
+  public biconomy: Biconomy
   public web3: Web3
+  public networkAgnostic: Web3
   public parentDefaultOptions: SendOptions
   public maticDefaultOptions: SendOptions
 
-  constructor(parentProvider, maticProvider, parentDefaultOptions, maticDefaultOptions) {
+  constructor(parentProvider, maticProvider, parentDefaultOptions, maticDefaultOptions, biconomyAPIKey?: string) {
     this.parentWeb3 = new Web3(parentProvider)
     this.web3 = new Web3(maticProvider)
+    this.biconomy = new Biconomy(new Web3.providers.HttpProvider(maticProvider), {
+      apiKey: biconomyAPIKey,
+      debug: true,
+    })
+    this.networkAgnostic = new Web3(this.biconomy)
```

Comment:
These steps only need to be done if `biconomyAPIKey` is provided, may throw error if it is undefined.

Reply:
gotcha, moved the biconomy web3 instance creation to network agnostic only. defining the biconomyAPIKey as send options in web3 client.